src/hooks/sync-sites/use-site-sync-management.ts

```diff
@@ -4,28 +4,82 @@ import { useAuth } from '../use-auth';
 import { SyncSite, useFetchWpComSites } from '../use-fetch-wpcom-sites';
 import { useSiteDetails } from '../use-site-details';
 
-const upToDateConnectedSites = (
+type UpToDateConnectedSitesReturn = {
+	updatedConnectedSites: SyncSite[];
+	toAdd: SyncSite[];
+	toDelete: { id: number; localSiteId: string }[];
+};
+
+export const upToDateConnectedSites = (
```

Comment:
Could we name it like a function to clearly say what it does, e.g., `reconcileConnectedSites` or `getConnectedSitesUpdates`? Then, returned type `ReconciledConnectedSites` or `ConnectedSitesUpdates`?

Reply:
`reconcileConnectedSites` sounds good, I will update it 👍

Reply:
I agree that "reconciled" seems like a good fit here 👍